docs/inputs.md

```diff
@@ -1260,6 +1260,30 @@ regional_pass: 10.90
     </tr>
 </table>
 
+<a id="cvm_establishment_synthesis"></a>
+
+### CVM Establishment Synthesis
+
+#### `percent_of_establishments_by_luz_size_emp_cat.csv`
+
+<table>
+    <tr>
+        <th>Sheet Name</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>Info</td>
+        <td>
+            Basic information of the data, and the size class definitions included
+        </td>
+    </tr>
+    <tr>
+        <td>Data</td>
+        <td>Detailed industry sector breakdown by size class and LUZ</td>
+    </tr>
+    
+</table>
+
```

Comment:
@mansisandag , there are more inputs for CVM (see input/CVM). Please add the following files and fields descriptions:

- MGRAEmpByEstSize.csv
- SummarySynthEstabs.csv
- SynthEstablishments.csv